src/Microsoft.Health.Fhir.TemplateManagement/Microsoft.Health.Fhir.TemplateManagement.csproj

```diff
@@ -15,7 +15,6 @@
     <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="3.1.9" />
     <PackageReference Include="Microsoft.Extensions.Options" Version="5.0.0" />
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
-    <PackageReference Include="RestSharp" Version="106.11.5" />
```

Comment:
maybe we should also remove the dependency from [nuget spec](https://github.com/microsoft/FHIR-Converter/blob/0a7363e512998f69a37d5484c8716ba7b0b6c236/src/Microsoft.Health.Fhir.TemplateManagement/Microsoft.Health.Fhir.Liquid.Converter.nuspec#L22)?

Reply:
Yes! It is removed in the update.